apps/maestro/src/features/InterchainTokenList/RegisteredInterchainTokenCard.tsx

```diff
@@ -185,7 +187,22 @@ export const RegisteredInterchainTokenCard: FC<Props> = (props) => {
         {!balance?.tokenBalance ? (
           !address ? null : (
             <div>
-              {isIncompatibleChain ? (
+              {props.chain?.chain_name?.toLowerCase().includes("stellar") ? (
```

Comment:
use id for consistency?
```suggestion
    {props.chain?.id?.toLowerCase().includes("stellar") ? (
```